runtime/vm/lookupmethod.c

```diff
@@ -1035,6 +1036,69 @@ illegalAccessMessage(J9VMThread *currentThread, IDATA badMemberModifier, J9Class
 	PORT_ACCESS_FROM_VMC(currentThread);
 	Trc_VM_illegalAccessMessage_Entry(currentThread, J9UTF8_LENGTH(senderClassNameUTF), J9UTF8_DATA(senderClassNameUTF),
 			J9UTF8_LENGTH(targetClassNameUTF), J9UTF8_DATA(targetClassNameUTF), badMemberModifier);
+
+#if defined(J9VM_OPT_VALHALLA_NESTMATES)
+	/* If an issue with the nest host loading and verification occurred, then it will be one of:
+	 * 		J9_VISIBILITY_NEST_HOST_LOADING_FAILURE_ERROR
+	 * 		J9_VISIBILITY_NEST_HOST_DIFFERENT_PACKAGE_ERROR
+	 * 		J9_VISIBILITY_NEST_MEMBER_NOT_CLAIMED_ERROR
+	 * Otherwise, it is one of:
+	 * 		J9_VISIBILITY_NON_MODULE_ACCESS_ERROR
+	 * 		J9_VISIBILITY_MODULE_READ_ACCESS_ERROR
+	 * 		J9_VISIBILITY_MODULE_PACKAGE_EXPORT_ERROR
+	 */
+	if ((J9_VISIBILITY_NEST_HOST_LOADING_FAILURE_ERROR == errorType)
+			|| (J9_VISIBILITY_NEST_HOST_DIFFERENT_PACKAGE_ERROR == errorType)
+			|| (J9_VISIBILITY_NEST_MEMBER_NOT_CLAIMED_ERROR == errorType)) {
```

Comment:
formatting nitpick: the closing `) {` should be on its own line at the same indent level as the `if`.  Makes it much easier to visually track the start / end of the if block.